client/assets/components/navbar.scss

```diff
@@ -0,0 +1,171 @@
+.top-navbar {
+  height: 70px;
+  border-radius: 0;
+  background-color: #fff;
+  margin: 0;
+  border: 0;
+  .navbar-brand {
+    height: 70px;
+    width: 70px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+}
+.navbar-collapse.collapse {
+  display: flex !important;
```

Comment:
### Critical

Is this being overridden by another style, if not, we don't need to use the `!important` tag. That should only be used when it's absolutely necessary.